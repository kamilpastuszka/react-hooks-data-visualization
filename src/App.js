import React, { Component } from "react";

class App extends Component {
  render() {
    return (
      <div className="container">
        <h2>Sample text</h2>
      </div>
    );
  }
}

export default App;
