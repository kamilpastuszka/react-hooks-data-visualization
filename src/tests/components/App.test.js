import React from "react";
import { render } from "enzyme";
import App from "../../App";

test("should render App component correctly", () => {
  const wrapper = render(<App />);
  expect(wrapper).toMatchSnapshot();
});
