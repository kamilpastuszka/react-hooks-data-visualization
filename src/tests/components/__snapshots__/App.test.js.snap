// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component correctly 1`] = `
<div
  class="container"
>
  <div>
    <div
      style="display:flex;justify-content:center"
    >
      <h2>
        Average temperature for London
      </h2>
    </div>
    <div
      class="recharts-wrapper"
      style="position:relative;cursor:default;width:730px;height:450px"
    >
      <svg
        class="recharts-surface"
        height="450"
        version="1.1"
        viewBox="0 0 730 450"
        width="730"
      >
        <clipPath
          id="recharts2-clip"
        >
          <rect
            height="385"
            width="620"
            x="80"
            y="30"
          />
        </clipPath>
        <g
          class="recharts-cartesian-grid"
        >
          <g
            class="recharts-cartesian-grid-horizontal"
          >
            <line
              fill="none"
              height="385"
              stroke="#ccc"
              stroke-dasharray="3 3"
              width="620"
              x="80"
              x1="80"
              x2="700"
              y="30"
              y1="30"
              y2="30"
            />
            <line
              fill="none"
              height="385"
              stroke="#ccc"
              stroke-dasharray="3 3"
              width="620"
              x="80"
              x1="80"
              x2="700"
              y="30"
              y1="415"
              y2="415"
            />
          </g>
          <g
            class="recharts-cartesian-grid-vertical"
          >
            <line
              fill="none"
              height="385"
              stroke="#ccc"
              stroke-dasharray="3 3"
              width="620"
              x="80"
              x1="80"
              x2="80"
              y="30"
              y1="30"
              y2="415"
            />
            <line
              fill="none"
              height="385"
              stroke="#ccc"
              stroke-dasharray="3 3"
              width="620"
              x="80"
              x1="700"
              x2="700"
              y="30"
              y1="30"
              y2="415"
            />
          </g>
        </g>
        <g
          class="recharts-layer recharts-cartesian-axis recharts-yAxis yAxis"
        >
          <line
            class="recharts-cartesian-axis-line"
            fill="none"
            height="385"
            stroke="#666"
            width="60"
            x="20"
            x1="80"
            x2="80"
            y="30"
            y1="30"
            y2="415"
          />
          <g
            class="recharts-cartesian-axis-ticks"
          />
        </g>
      </svg>
      <div
        class="recharts-tooltip-wrapper"
        style="pointer-events:none;visibility:hidden;position:absolute;top:0;transform:translate(undefinedpx, undefinedpx);-webkit-transform:translate(undefinedpx, undefinedpx);-moz-transform:translate(undefinedpx, undefinedpx);-o-transform:translate(undefinedpx, undefinedpx);-ms-transform:translate(undefinedpx, undefinedpx)"
      >
        <div
          class="recharts-default-tooltip"
          style="margin:0;padding:10px;background-color:#fff;border:1px solid #ccc;white-space:nowrap"
        >
          <p
            class="recharts-tooltip-label"
            style="margin:0"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
